.github/workflows/docker.yml

```diff
@@ -523,8 +523,6 @@ jobs:
     uses: ./.github/workflows/docker-wiki-update.yml
     needs: [aarch64-images-tag-push-fast, x86_64-images-tag-push-fast]
     if: contains(github.event.pull_request.title, '[FAST_BUILD]')
-    permissions:
-      contents: write
```

Comment:
I don't think this is needed, because we only change wiki in `main`, and this won't run in `main`